src/screens/SettingsScreen.js

```diff
@@ -132,6 +140,42 @@ export const SettingsScreen = () => {
         });
       }
 
+      if (settings.onboarding) {
+        const onboarding = await readFromStore(ONBOARDING_STORE_KEY);
+
+        additionalSectionedData.push({
+          data: [
+            {
+              title: texts.settingsTitles.onboarding,
+              topDivider: false,
+              value: onboarding === 'complete' ? false : true,
```

Comment:
`onboarding === 'complete'` this expression already returns `true` or `false`, which is not needed to do explicitly.
i would suggest `value: onboarding === 'complete'` here

Reply:
a7587b4e5e398ac83c0cd01a398797166b4d72b2